cllc-public-app/ClientApp/src/app/components/sep/my-applications/submitted-applications/submitted-applications.component.html

```diff
@@ -60,18 +70,22 @@
         <!-- Actions -->
         <ng-container matColumnDef="actions">
           <th mat-header-cell *matHeaderCellDef>Action</th>
-          <td mat-cell *matCellDef="let row" class="p-0">
+          <td mat-cell *matCellDef="let row" class="p-0" style="padding: 10px !important; ">
```

Comment:
class p-0 here sets the padding to 0. That is probably why padding: 10px did not work without the !important. Removing the class here would be a good idea.